joomla/joomla-cms
# Hathor: missing new custom fields menus + CS + generatecss.php

The new custom fields menus (for Users and Content) were not implemented in Hathor
### Summary of Changes

Adding the necessary code for Hathor mod_menu
Added a new css in Hathor to get an image for the Fields menu.

**NOTE:**
As I had a new css for Hathor, I ran generatecss.php.
It apparenty had not been running for ages as it corrected a lot of css stuff unrelated to this PR.
Most of it concerns spaces changed to tabs (as should), but it also took off some obsolete bootstrap-extended css, based on the bootstrap-extended.less file.
### Testing Instructions

Make sure you implement Custom fields in Users and Content manager Options and switch template to Hathor.
Patch and look again

![screen shot 2016-10-30 at 10 06 09](https://cloud.githubusercontent.com/assets/869724/19835557/e0397082-9e8a-11e6-8ded-6591b963b607.png)
![screen shot 2016-10-30 at 10 06 01](https://cloud.githubusercontent.com/assets/869724/19835558/e03aeea8-9e8a-11e6-8895-84a9ebc69650.png)
